tests/test_collection.py

```diff
@@ -0,0 +1,54 @@
+from .test_base import TestBase
+from django.urls import reverse
+
+
+class CollectionTest(TestBase):
+    def test_logged_in_user_can_post_collection(self):
+        # log in as owner
+        self.client.login(username="owner", password="password123")
+        # try to post a collection
+        response = self.client.post(reverse('collection-list'),
+                                    {'title': 't', 'description': 'd', 'source': 'mst3k', 'tags': ''})
+        # assert success
```

Comment:
This is more of a personal preference, but I think these assert statements are simple enough as to not need comments